static/directives/repository-events-table.html

```diff
@@ -6,7 +6,7 @@
       <div class="heading-controls hidden-sm hidden-xs">
         <a href="/repository/{{ repository.namespace }}/{{ repository.name }}/create-notification"
            class="btn btn-primary" ng-show="repository.can_write && !inReadOnlyMode">
-          <i class="fa fa-plus"></i> Create Notification
+           <span class=""><i class="fa fa-plus"></i> Create Notification</span>
```

Comment:
remove `class=""`

Reply:
I added the class back in to fix the [issue above](https://github.com/quay/quay/pull/34#discussion_r349799465).